spectator-ext-jvm/src/test/java17/com/netflix/spectator/jvm/JavaFlightRecorderTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2014-2024 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spectator.jvm;
+
+import com.netflix.spectator.api.DefaultRegistry;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Measurement;
+import com.netflix.spectator.api.Registry;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JavaFlightRecorderTest {
+
+  @Test
+  public void isSupported() {
+    assertTrue(JavaFlightRecorder.isSupported());
+  }
+
+  @Test
+  public void checkDefaultMeasures() throws Exception {
+    Registry registry = new DefaultRegistry();
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    try (var closable = JavaFlightRecorder.monitorDefaultEvents(registry, executor)) {
+      Thread.sleep(6000);
```

Comment:
Was this chosen just to be longer than the 5s period used on the `EventSettings`?

Reply:
That's right.